packages/clerk-js/src/ui/Components.tsx

```diff
@@ -281,6 +294,32 @@ const Components = (props: ComponentsProps) => {
     </AppearanceProvider>
   );
 
+  const mountedCreateOrganizationModal = (
+    <AppearanceProvider
+      globalAppearance={state.appearance}
+      appearanceKey='createOrganization'
+      appearance={createOrganizationModal?.appearance}
+    >
+      <FlowMetadataProvider flow='createOrganization'>
+        <InternalThemeProvider>
+          <Modal
+            handleClose={() => componentsControls.closeModal('createOrganization')}
+            containerSx={{ alignItems: 'center' }}
+            contentSx={t => ({ height: `min(${t.sizes.$120}, calc(100% - ${t.sizes.$12}))`, margin: 0 })}
+          >
+            <VirtualRouter
+              preservedParams={PRESERVED_QUERYSTRING_PARAMS}
+              onExternalNavigate={() => componentsControls.closeModal('createOrganization')}
+              startPath='/createOrganization'
```

Comment:
:question: Is this the default value for `createOrganizationUrl`? I thought it was `/create-organization`. Or is this something different?

Reply:
Hey Gianni, this is a path used by the virtual router the modal components use by default. This is an implementation detail of the virtual router and not something visible to the end developer/ user